include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -522,6 +522,7 @@ struct __usm_or_buffer_accessor
 struct __result_and_scratch_storage_base
 {
     virtual ~__result_and_scratch_storage_base() = default;
+    virtual std::size_t __get_data(sycl::event, std::size_t* __p_buf) const = 0;
```

Comment:
As far as `__result_and_scratch_storage_base` already has `__` in their  name, I believe additional `__` isn't required in method name.

Reply:
Basically, we add underscore for the all internal names - types, aliases, variables, methods, macros etc.